Detectors/FIT/base/include/FITBase/Geometry.h

```diff
@@ -0,0 +1,88 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_FIT_GEOMETRY_H_
+#define ALICEO2_FIT_GEOMETRY_H_
+////////////////////////////////////////////////
+// Full geomrtry  hits classes for detector: FIT    //
+////////////////////////////////////////////////
+
+#include <Rtypes.h>
+#include <TNamed.h>
+namespace o2
+{
+namespace FIT
+{
+
+class Geometry : public TNamed
+{
+ public:
+   ///
+  /// Default constructor.
+  /// It must be kept public for root persistency purposes,
+  /// but should never be called by the outside world
+  Geometry() = default;
+
+  ///
+  /// Constructor for normal use.
+  ///
+  ///
+  Geometry(const Text_t* name, const Text_t* title = "", const Text_t* mcname = "", const Text_t* mctitle = "");
+
+  ///
+  /// Copy constructor.
+  ///
+  Geometry(const Geometry& geom);
+
+  ///
+  /// Destructor.
+  ///
+  ~Geometry() override ;
+
+  ///
+  /// Assign operator.
+  ///
+  Geometry& operator=(const Geometry& rvalue);
```

Comment:
sigletons should not have copy + assignment operators. These should just be deleted using `Geometry(const Geometry& geom) = delete;`

Reply:
is Geometry really a singleton ?

Reply:
Not sure but why else would it have a `static GetInstance()` function?